docs/developer-docs/latest/development/custom-fields.md

```diff
@@ -198,6 +223,47 @@ Each object in the `items` array can contain the following parameters:
 | `intlLabel`     | Translation for the label of the input                             | [`IntlObject`](https://formatjs.io/docs/react-intl/) |
 | `type`          | Type of the input (e.g., `select`, `checkbox`)                     | `String`                                             |
 
+::: details "type" accepts the following inputs:
+
+```javascript
+[
+  'text',
+  'checkbox',
+  'checkbox-with-number-field',
+  'boolean-radio-group',
+  'select-number',
+  'radio-group',
+  'select',
+  'select-default-boolean',
+  'select-date',
+  'textarea-enum',	
+]
```

Comment:
We should probably test all of these and commit them as examples on a file in the examples/getstarted, similar to how we do for types:

https://github.com/strapi/strapi/blob/3a33df8e7e53547d82d4fb65e9fa8395a01b8b7b/examples/getstarted/src/plugins/myplugin/admin/src/index.js#L71

That could be something I do though if you don't have the time.